wandb/wandb_torch.py

```diff
@@ -299,18 +316,58 @@ def _torch_hook_handle_is_valid(self, handle):
         else:
             return handle.id in d
 
-    def _no_finite_values(self, tensor: "torch.Tensor") -> bool:
+    def _is_valid_mask(self, tensor: "torch.Tensor") -> bool:
+        if tensor.dtype == torch.float32:
+            high = HIGH32
+        else:  # dtype == torch.float64
```

Comment:
is this a safe assumption given that types can be anything?
(maybe mathematically it works out?)